Docker building error

👋  I am following the instructions in README to build and run tests in a container, however during the build process (master branch) I am getting :

```
Use --sandbox_debug to see verbose messages from the sandbox
In file included from external/arrow/cpp/src/arrow/vendored/datetime.h:20:0,
                 from external/arrow/cpp/src/arrow/util/parsing.h:34,
                 from external/arrow/cpp/src/arrow/ipc/json_simple.cc:33:
external/arrow/cpp/src/arrow/vendored/datetime/date.h: In function 'constexpr std::chrono::hours arrow_vendored::date::make12(std::chrono::hours)':
external/arrow/cpp/src/arrow/vendored/datetime/date.h:3936:15: error: call to non-constexpr function 'std::chrono::duration<_Rep, _Period>& std::chrono::duration<_Rep, _Period>::operator-=(const std::chrono::duration<_Rep, _Period>&) [with _Rep = long int; _Period = std::ratio<3600l>]'
             h -= hours{12};
               ^
external/arrow/cpp/src/arrow/vendored/datetime/date.h: In function 'constexpr std::chrono::hours arrow_vendored::date::make24(std::chrono::hours, bool)':
external/arrow/cpp/src/arrow/vendored/datetime/date.h:3950:15: error: call to non-constexpr function 'std::chrono::duration<_Rep, _Period>& std::chrono::duration<_Rep, _Period>::operator+=(const std::chrono::duration<_Rep, _Period>&) [with _Rep = long int; _Period = std::ratio<3600l>]'
             h += hours{12};
```

the command I am running to build is as mentioned in the docs
````
bazel build -c opt --copt=-march=native --copt=-fPIC -s --verbose_failures //tensorflow_io/...
```

--- a/tools/dev/Dockerfile
+++ b/tools/dev/Dockerfile
@@ -1,6 +1,6 @@
 FROM tensorflow/tensorflow:custom-op-ubuntu16
 
-RUN rm /etc/apt/sources.list.d/jonathonf-ubuntu-python-3_6-xenial.list && apt-get update && \
+RUN rm -f /etc/apt/sources.list.d/jonathonf-ubuntu-python-3_6-xenial.list && apt-get update && \
     apt-get install -y \
     git \
     gcc \

